EIPS/eip-3643.md

```diff
@@ -375,7 +356,9 @@ The standard is implemented and tested with full coverage on Tokeny's [T-REX rep
 
 ## Security Considerations
 
-The suite of Smart Contracts has been audited by an external and independent company. The results can be found in [this document](https://tokeny.com/wp-content/uploads/2020/05/Tokeny-Solutions_T-REX-v3_Smart-Contract-Audit-Report_Kapersky.pdf).
+The suite of Smart Contracts has been audited by external independent companies. The results can be found here :
+- [T-REX v3](https://tokeny.com/wp-content/uploads/2020/05/Tokeny-Solutions_T-REX-v3_Smart-Contract-Audit-Report_Kapersky.pdf).
+- [T-REX v4](https://tokeny.com/wp-content/uploads/2023/04/Tokeny_TREX-v4_SC_Audit_Report.pdf)
```

Comment:
You should add the files to the assets folders directly. I think it is not possible anymore to add a non-relative link on EIPs